### ISSUE TITLE
math.inv() causes "Can not calculate inverse, determinant is zero" 

### ISSUE THREAD

**Issue by hogegex** [josdejong/mathjs#1122]
The following simple code works fine with mathjs v4.4.0, but it throws an exception in v4.4.1.

```javascript
let mat = math.matrix ([
   [1, 0, 0],
   [0, -1, 1],
   [0, 0, 1]
]);

console.log (math.inv (mat)); // Error: Can not calculate inverse, determinant is zero
```

This is probably a bug caused by #1114.

**Comment by ericman314** [comment]
Sorry about that, I'll try to get it fixed tonight.

**Comment by ericman314** [comment]
Fixed in the develop branch via #1124.

**Comment by josdejong** [comment]
Thanks a lot for the quick fix Eric! The fix is applied now in `v4.4.2`
